core/src/androidTest/java/org/hisp/dhis/android/core/organisationunit/internal/OrganisationUnitCallMockIntegrationShould.kt

```diff
@@ -68,7 +69,7 @@ class OrganisationUnitCallMockIntegrationShould : BaseMockIntegrationTestEmptyEn
         val organisationUnits = listOf(orgUnit)
 
         // dependencies for the OrganisationUnitCall:
-        val organisationUnitService = OrganisationUnitService(d2.httpServiceClient())
+        val organisationUnitNetworkHandler: OrganisationUnitNetworkHandler = DhisAndroidSdkKoinContext.koin.get()
```

Comment:
Useful way to get objects from the koin tree, so they don't have to have initialized in the tests. A lot of code could be replaced by this way of getting objects.